source/docs/steeltoe-configuration.html.markdown

```diff
@@ -420,18 +456,20 @@ On a Linux cell, you should see something like this during startup. On Windows c
 
 ```
 
-#### 2.1.2.5 What to expect
+On Windows cells, you should see something slightly different but with the same content.
 
-The `cf push` will create an app by the name `foo` and will bind the `myConfigServer` service instance to the app. You can hit the app at `http://foo.x.y.z/`.
+#### 2.1.2.5 View Results
+
+The `cf push` command creates an application by the name of `sample` and binds the `myConfigServer` service instance to the application. You can see the application at `http://sample.x.y.z/`.
 
 Use the menu provided by the app to see various output related to Cloud Foundry and the config server:
 
-* `CloudFoundry Settings` - should show `VCAP_APPLICATION` and `VCAP_SERVICES` settings read as configuration data.
-* `Config Server Settings` - should show the settings used by the Steeltoe client when communicating to the Config Server.  These have been picked up from the service binding.
-* `Config Server Data` - this is the configuration data returned from the Config Servers configured github repository. It will be some of the data from `foo.properties`, `foo-development.properties` and `application.yml` found in the github repository (<https://github.com/spring-cloud-samples/config-repo>).
-* `Reload` - will cause a reload of the configuration data from the Config Server.
+* `CloudFoundry Settings`: Should show `VCAP_APPLICATION` and `VCAP_SERVICES` settings read as configuration data.
+* `Config Server Settings`: Should show the settings used by the Steeltoe client when communicating to the Config Server. These have been picked up from the service binding.
+* `Config Server Data`: This is the configuration data returned from the Config Server's configured GitHub repository. It includes some of the data from `sample.properties`, `sample-development.properties` and `application.yml` found in the GitHub repository: (<https://github.com/spring-cloud-samples/config-repo>).
```

Comment:
All Changes from 'foo' to 'sample' need to be reversed or change the sample:
https://github.com/spring-cloud-samples/config-repo